console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -1794,6 +1794,60 @@ function makeAddressSpaceMetrics(as) {
   ];
 }
 
+function makeIotProjectMetrics(pj) {
+  var devices =
+    iotdevices[getIotDevicesIndexForProjectName(pj.metadata.name)].devices;
+  var cons =
+    as.metadata.uid in addressspace_connection
+      ? addressspace_connection[as.metadata.uid]
+      : [];
+
+  return [
+    {
+      name: "Max connection",
+      type: "counter",
+      value: cons.length,
+      units: "connections"
+    },
+    {
+      name: "Data volume",
+      type: "counter",
+      value: devices.length * 1000,
+      units: "bytes"
+    },
+    {
+      name: "Start date",
+      type: "counter",
+      value: getRandomCreationDate(),
+      units: "Date"
+    },
+    {
+      name: "End date",
+      type: "counter",
+      value: getRandomCreationDate(),
+      units: "Date"
+    },
```

Comment:
@k-wall As I stated on the previous PR : 

what I mean is that in the Metric garphQL object the `value` field is typed as a float : 

https://github.com/EnMasseProject/enmasse/blob/2f1c035b1118d825dc66a8ee8e859e5ab0471315/console/console-server/src/main/resources/schema.graphql#L99-L104

Should I add a string type there ?